src/transforms/transform-code-scanning-alert.ts

```diff
@@ -101,3 +109,66 @@ export const transformCodeScanningAlert = async (context: WebhookContext, github
 		}]
 	};
 };
+
+const transformRuleTagsToIdentifiers = (tags: string[] | null) => {
+	if (!tags) {
+		return null;
+	}
+	// CWE tags from GitHub take the format 'external/cwe/cwe-259'
+	const cwePrefix = "external/cwe/cwe-";
+	const identifiers = tags.filter(tag => tag.startsWith(cwePrefix)).map(tag => {
+		// Remove starting 0s as GitHub can provide 'cwe-079'
+		const cweId = tag.split(cwePrefix)[1].replace(/^0+/, "");
+		return {
+			displayName: `CWE-${cweId}`,
+			url: `https://cwe.mitre.org/cgi-bin/jumpmenu.cgi?id=${cweId}`
```

Comment:
What's the difference between using this URL and `https://cwe.mitre.org/data/definitions/${cweId}` ? 
The one you are using redirects to the `data/definitions` page anyway?

Reply:
I used the URL https://cwe.mitre.org/ uses in their ID Lookup input. The difference is the error page when supplying an invalid ID. I suspect there is future-proof credence using the `cgi` URL as it would be out in the wild and reference in PDFs, hence the redirect.

Navigating using the definitions URL we get:
<img width="677" alt="Screenshot 2023-08-15 at 1 27 31 pm" src="https://github.com/atlassian/github-for-jira/assets/1657832/443dd4bd-da12-4b45-b21d-1adf33b2a8b2">

Whereas with the `id` param, we get:
<img width="740" alt="Screenshot 2023-08-15 at 1 27 37 pm" src="https://github.com/atlassian/github-for-jira/assets/1657832/35b96fce-9dc3-4dab-a43b-e7b470e7b839">